### ISSUE TITLE
GZIP support

### ISSUE THREAD

**Issue by OlafvdSpek** [svanoort/pyresttest#239]
pyresttest doesn't appear to support GZIP requests / responses yet, could this be supported?

**Comment by postmart** [comment]
did you try to enable gzip first?
```
- test:
    - curl_option_encoding: "gzip"
```

**Comment by OlafvdSpek** [comment]
No. Why isn't it enabled by default? And how do I enable this for all tests?

**Comment by postmart** [comment]
To enable for all the tests put it under the config part.
```
---
- config:
```
Can you verify whether it worked for you?

**Comment by OlafvdSpek** [comment]
Neither option appears to work.
